src/data/downloads.tsx

```diff
@@ -151,6 +151,55 @@ sudo apt install jellyfin`}
       }
     ]
   },
+  {
+    id: 'flatpak',
+    name: 'Flatpak',
+    osTypes: [OsType.Linux],
+    status: DownloadStatus.Community,
+    features: [Feature.CustomFFmpeg],
+    platforms: [Platform.Linux],
+    description: 'Install Jellyfin via Flathub.',
+    stableButtons: [
+      {
+        id: 'flatpak-stable-button',
+        name: 'Install Instructions',
+        details: (
+          <>
+            <p>
+              <b>Note:</b> Make sure Flatpak and Flathub are already setup before installing.
```

Comment:
This note seems a bit out of place, why would someone explicitly opening the flatpak instructions not have flatpak installed/setup?